index.bs

```diff
@@ -2199,7 +2222,7 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
     [[#sctn-make-credential-privacy]] for details.
 
 During the above process, the user agent SHOULD show some UI to the user to guide them in the process of selecting and
-authorizing an authenticator.
+authorizing an authenticator. When <code>|options|.{{CredentialCreationOptions/mediation}}</code> is set to {{CredentialMediationRequirement/conditional}}, prominent modal UI should <i>not</i> be shown <i>unless</i> credential creation was previously consented to via means determined by the user agent.
```

Comment:
What should be the UP and UV flags for the response returned in this silent operation? If UP is zero, then current spec does not allow that and we may need to see that section. Or are we saying that UP is 1, in this conditional create?

Reply:
We would report UP=1 for conditional create. UV depends on the specifics of the authenticator. Some might be able to collect UV, others might not and would be unable to satisfy requests with UV=required.

Reply:
Typical authenticator have no insight to what has happened in the browser layer. As the operation is silent from authenticator's boundary perspective, UP=0 and UV=0 makes more sense to me. 

This is an opt-in behavior so RP can make changes to expect such a response.

Reply:
As an RP developer, I'd like to request _"I want UV on authentication, but no need at registration timing"_, and want authenticator answer _"OK, I didn't do UV on registration, but I ensure that you can request UV on following authentications"_.
Does webauthn has such mechanism now?

Reply:
@nov, creation and authentication are different ceremonies. You as the RP are in control of whether UV is required or not for each ceremony.

Reply:
I don't want users to register passkeys which cannot fulfill UV requirement on future authentication.

Reply:
> I don't want users to register passkeys which cannot fulfill UV requirement on future authentication.

Then you need to use UV=required, and this feature won't be applicable to your environment.